src/gui/mod.rs

```diff
@@ -1987,7 +1991,7 @@ async fn load_caches() -> Result<(FingerprintCache, AddonCache)> {
     Ok((fingerprint_cache, addon_cache))
 }
 
-fn apply_config(ajour: &mut Ajour, config: Config) {
+pub fn apply_config(ajour: &mut Ajour, config: Config) {
```

Comment:
How do you like this, @tarkah? I would need to access `apply_config` from `update`.

Reply:
Yeah I think that makes sense! No reason to reinvent the wheel